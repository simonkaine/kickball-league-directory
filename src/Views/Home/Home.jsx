export function Home() {
    return (
        <h1>
            WHAT IT DO BABY BOO
        </h1>
    )
}