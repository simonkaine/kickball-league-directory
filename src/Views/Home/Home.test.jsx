import { render, screen } from "@testing-library/react";
import { Home } from "./Home";

it('should test for home view render', () => {
    const { container } = render (
        <Home />
    )

    screen.getByText('Welcome to the Kickball League Directory!');
    expect(container).toMatchSnapshot();
})