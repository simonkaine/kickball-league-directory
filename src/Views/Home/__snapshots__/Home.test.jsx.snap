// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test for home view render 1`] = `
<div>
  <section
    class="HomePage"
  >
    <div
      style="height: 100vh; margin: 100px 0px 0px 0px; background-color: rgb(255, 255, 240);"
    >
      <h1
        style="font-size: 3em; margin: -50px 0px 50px 0px;"
      >
        Welcome to the Kickball League Directory!
      </h1>
      <img
        alt="Timbers Stadium"
        src="kickball.jpg"
      />
    </div>
  </section>
</div>
`;
